src/IssueCreator/Dialogs/ManageIssue.cs

```diff
@@ -88,6 +71,13 @@ private async Task ExecuteIssueOperation(Func<long, int, long, int, Task<bool>>
             }
         }
 
+        private async Task<bool> ExecuteIssueOperationInternal(long repoId, Func<long, int, long, int, Task<bool>> action)
+        {
+            IssueDescription epicInfo = cboEpics.SelectedItem as IssueDescription;
+
+            return await action(epicInfo.Repo.Id, epicInfo.Issue.Number, repoId, int.Parse(txtIssueNumber.Text));
```

Comment:
Probably never going to happen, but I would check that `epicInfo` is not null here.

Reply:
And I would say similar things around the `Parse` method as above.